shuup_tests/admin/test_shipment_creator.py

```diff
@@ -165,3 +166,24 @@ def form_valid_hook(self, form, shipment):
             shipping_data[shipment.identifier] = {"phone": data.get("phone")}
             shipment.order.shipping_data = shipping_data
             shipment.order.save()
+
+
+@pytest.mark.django_db
+def test_shipment_creating_with_no_shipping_address(rf, admin_user):
+    shop = get_default_shop()
+    supplier = get_default_supplier()
+    product = create_product(sku="test-sku", shop=shop, supplier=supplier, default_price=3.33)
+    order = create_order_with_product(product, supplier, quantity=1, taxless_base_unit_price=1, shop=shop)
+    
+    # remove shipping address 
+    order.shipping_address = None
+    order.save()
+
+    try:
```

Comment:
Any chance we can hide the "create shipment" toolbar button if there is no shipping address? Shipping addresses shouldn't be required for digital goods/marketplace items.